src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  position: relative;
+  margin: 20px 8px;
+}
+
+.search-bar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.big-bar {
+  padding-left: 62px;
+  height: 70px;
+}
```

Comment:
please, fix your classes according to checklist, point 2
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md